packages/test/test-end-to-end-tests/src/test/summarizerWithLocalChanges.spec.ts

```diff
@@ -313,6 +321,120 @@ describeNoCompat("Summarizer with local data stores", (getTestObjectProvider) =>
 		},
 	);
 
+	itExpects(
+		"ValidateSummaryBeforeUpload = true. Summary should fail if ops are sent before summarize",
+		[
+			{
+				eventName: "fluid:telemetry:Summarizer:Running:Summarize_cancel",
+				clientType: "noninteractive/summarizer",
+				error: "PendingOpsWhileSummarizing",
+				beforeGenerate: true,
+			},
+		],
+		async () => {
+			// Wait for 100 ms for pending ops to be saved.
+			const pendingOpsTimeout = 100;
+			settings["Fluid.ContainerRuntime.SubmitSummary.waitForPendingOpsTimeout"] =
+				pendingOpsTimeout;
+			const mockLogger = new MockLogger();
+			const container1 = await provider.makeTestContainer();
+			const { summarizer, container: summarizerContainer } = await createSummarizer(
+				provider,
+				container1,
+				{ loaderProps: { configProvider: mockConfigProvider(settings) } },
+				undefined /* summaryVersion */,
+				mockLogger,
+			);
+
+			const defaultDataStore1 = await requestFluidObject<ITestDataObject>(
```

Comment:
nit: entryPoint?

Reply:
Can't use it here because the test runtime factory doesn't pass initialize entry point yet.

Reply:
k. I can't keep track of where it is and isn't present yet in the test infra so sorry if I keep asking :P

Reply:
Haha yeah its confusing. I would honestly not worry about it here and do all of them when request pattern is completely removed. The tests will have to be updated then.